release/air_tests/air_benchmarks/workloads/xgboost_benchmark.py

```diff
@@ -107,10 +114,17 @@ def run_xgboost_prediction(model_path: str, data_path: str):
 
 
 def main(args):
-    experiment_params = _EXPERIMENT_PARAMS[args.size]
+
+    if args.smoke_test:
+        experiment_params = _EXPERIMENT_PARAMS["smoke_test"]
+        cpus_per_worker = 1
```

Comment:
Move `cpus_per_worker` into `_EXPERIMENT_PARAMS` so there's one source of conifgs?